LocalSettings.php

```diff
@@ -2489,12 +2489,18 @@
 			'sysadmin' => [
 				'globalgroupmembership' => true,
 				'globalgrouppermissions' => true,
+				'oathauth-api-all' => true,
```

Comment:
Drop altogether please

Reply:
Could also be replaced with oathauth-verify-user which allows the same thing to be done.

Reply:
Yep.

Reply:
I don't see any harm in leaving `oauth-api-all` local. It's a local user right for SRE and Trust and Safety online.  As for `oathauth-verify-user`, this exists in the global `sysadmin` and `trustandsafety` global groups, which I'd prefer to leave as global, should Meta Wiki be inaccessible in some way.

Reply:
I am asking you as a member of SRE to remove it. I'd rather SRE only have verify-user on meta and not api-all at all.

Reply:
If metawiki is not accessible then more then likely many others are inaccessible as well and the higher priority would be to restore access. For logging reasons it should never be used on other wikis anyway.

Reply:
> I am asking you as a member of SRE to remove it. I'd rather SRE only have verify-user on meta and not api-all at all. If metawiki is inaccessible and we need this urgently, we have a DB.

Oh, sorry, I thought you were replying about both groups. Yeah, I guess we don't really need `oathauth-api-all` and if you guys prefer `oathauth-verify-user` only Meta Wiki, that would be fine

Reply:
We would. Can you make the change here?

Reply:
Yes, doing this now.